tests/integration/questionnaire/test_questionnaire_relationships.py

```diff
@@ -18,23 +18,23 @@ def test_valid_relationship(self):
         self.post()
         self.assertInUrl("/questionnaire/sections/")
 
-    def test_get_first_relationship_with_resume(self):
+    def test_resume_should_not_show_last_viewed_guidance(self):
         self.launchSurvey("test_relationships")
         first_list_item_id = self.add_person("Marie", "Doe")
         second_list_item_id = self.add_person("John", "Doe")
         self.post({"anyone-else": "No"})
         self.get("/questionnaire/relationships?resume=True")
         self.assertInUrl(
-            f"/questionnaire/relationships/people/{first_list_item_id}/to/{second_list_item_id}/?resume=True"
+            f"/questionnaire/relationships/people/{first_list_item_id}/to/{second_list_item_id}"
```

Comment:
This assertion is true with or without resuming, it should assert instead that `resume=True` is not in the url.

Reply:
I yeah, lol